src/components/docs/WaveBackground/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+const WaveBackground = () => (
+  <div className={styles.root}>
+    <div>
+      <img src="/img/docs/wave-top.png" />
```

Comment:
This asset needs to be transparent. If the background color changes (or dark mode is enabled) it sticks out.